pytype/abstract/class_mixin.py

```diff
@@ -579,8 +578,8 @@ def _get_attrs_from_mro(self, cls_attrs, metadata_key):
       # Any subclass of a Parameterized dataclass must inherit attributes from
       # its parent's init.
       # See https://github.com/google/pytype/issues/1104
-      if isinstance(base_cls, _classes.ParameterizedClass):
-        type_params = base_cls._formal_type_parameters
+      if abstract_utils._isinstance(base_cls, "ParameterizedClass"):
```

Comment:
can just be `if _isinstance(...)`, we have it aliased somewhere near the top of the file.